RELEASES.md

```diff
@@ -4,68 +4,72 @@
 
 Due to the period of time between this release and v0.0.26, many relevant
 changes and fixes may not appear in the following list.  For a complete list of
-changes, please check: https://github.com/google/kythe/commits/
+changes, please check the commit logs:
+https://github.com/google/kythe/search?q=committer-date%3A2016-11-01..2018-05-31&type=Commits
 
-Notable changes:
- - remove GRPC server and client code
+### Notable additions
+
+ - First release of Go indexer and Go extractors.
+ - Objective C is now supported in the C++ indexer.
+ - First release of TypeScript indexer.
```

Comment:
This is not included in the release.

Reply:
Should it be?

Reply:
I was pushing it the next release to simplify.

Reply:
Makes sense—I removed it.